.github/workflows/_build_test_upload.yml

```diff
@@ -34,12 +34,10 @@ jobs:
     steps:
       - name: Get Release Type
         run: |
-          if [[ ${{ inputs.branch }} == release/* ]]; then
-            if [[ ${{ inputs.pre_dev_release }} == true ]]; then
-              RELEASE_TYPE=test
-            else
-              RELEASE_TYPE=official
-            fi
+          if [[ ${{ inputs.branch }} == v* ]] && [[ ${{ inputs.pre_dev_release }} == false ]]; then
```

Comment:
Should this be `release/*` instead of `v*`?

Reply:
For official release, we should use tags rather than release branch. In order to retrieve tags, the input branch for official release has to be something like `v0.3.0`.